dashboards/google-vertex-ai/vertex-ai-custom-training.json

```diff
@@ -0,0 +1,344 @@
+{
+  "displayName": "Vertex AI Custom Training Dashboard",
+  "dashboardFilters": [
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "job_id",
+      "stringValue": "",
+      "templateVariable": "",
+      "valueType": "STRING"
+    },
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "region",
+      "stringValue": "",
+      "templateVariable": "",
+      "valueType": "STRING"
+    }
+  ],
+  "labels": {},
+  "mosaicLayout": {
+    "columns": 48,
+    "tiles": [
+      {
+        "height": 16,
+        "width": 48,
+        "widget": {
+          "title": "CPU and Memory",
+          "collapsibleGroup": {
+            "collapsed": false
+          },
+          "id": ""
+        }
+      },
+      {
+        "xPos": 1,
+        "height": 16,
+        "width": 22,
+        "widget": {
+          "title": "Cloud ML Job - CPU utilization [SUM]",
```

Comment:
now that it's grouped by job_id I'd suggest removing the auto-generated "[SUM]" from the titles. I think that probably is more confusing than helpful now